app/code/community/Catalin/SEO/controllers/CategoryController.php

```diff
@@ -90,7 +90,8 @@ public function viewAction()
 
                 $response = array(
                     'listing' => $listing,
-                    'layer' => $layer
+                    'layer' => $layer,
+                    'subheading' => Mage::helper('flaman_productlocations')->getFilterLocationName(),
```

Comment:
This helper is not in the default magento installation. I can't accept dependencies on other extensions.